lnbits/static/js/event-reactions.js

```diff
@@ -1,16 +1,30 @@
-function eventReactionWebocket(event_id, event_function) {
+function eventReactionWebocket(event_id) {
   localUrl = ''
+  reaction = localStorage.getItem('lnbits.reactions')
+  if (!reaction || reaction === 'None') {
+    return
+  }
   if (location.protocol !== 'http:') {
     localUrl = 'wss://' + location.host + '/api/v1/ws/' + event_id
   } else {
     localUrl = 'ws://' + location.host + '/api/v1/ws/' + event_id
   }
   connection = new WebSocket(localUrl)
```

Comment:
when is this websocket closed?

Suggestion: close if the expected message is received or after a time interval.